src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,37 @@
+package duke.task;
+
+import duke.task.Task;
+
+public class Deadline extends Task {
+
+    protected String by;
+
+    public static final String DEADLINE_LABEL = "[D]";
```

Comment:
Can the scope of this variable be reduced? Similar changes can be made in other places as well.

Reply:
Which variable do you mean? by or DEADLINE_LABEL?

Reply:
DEADLINE_LABEL